brefv/payloads/foxglove/Imu.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package foxglove;
+
+import "foxglove/Quaternion.proto";
+import "foxglove/Vector3.proto";
+import "foxglove/Header.proto";
+
+message Imu {
+  Header header = 1;
```

Comment:
I suggest to stick with the same strategy that is used in the foxglove package. Instead of the Header, we use a `google.protobuf.Timestamp` and `frame_id` explicitly in the `ImuReading` message. Compare with: https://github.com/MO-RISE/keelson/blob/main/brefv/payloads/foxglove/PointCloud.proto#L13-L17

Reply:
Fixed